setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as readme_file:
+    readme = readme_file.read()
+
+requirements = ["casbin==0.8.4", "psycopg2-binary==2.8.6", "black==20.8b1"]
+
+setup(
+    name="postgresql-watcher",
+    version="0.0.1",

```

Comment:
Once @Zxilly completes the CI in the other PR,  we can change the version through the CI for every release.